site/content/in-dev/unreleased/getting-started/using-polaris/keycloak-idp.md

```diff
@@ -48,7 +50,7 @@ Polaris is configured with 3 realms:
   issued by both Polaris and Keycloak.
 
 For more information about how to configure Polaris with external authentication, see the
-[Polaris documentation]({{% ref "../external-idp" %}}).
+[Polaris documentation](https://polaris.apache.org/in-dev/unreleased/managing-security/external-idp/).
```

Comment:
Can this be a "hugo" `ref`? Docs will be "frozen" after a release to links to `in-dev` from old versions are likely to become invalid pretty soon.